src/main.js

```diff
@@ -50,14 +50,48 @@ const redirect = url => {
   }
 }
 
+const defaultDownloadLinks = {
+  linux_link:
+    "https://github.com/inexorgame/inexor-core/releases/download/0.8.10-alpha/Inexor-0.8.10-alpha-Linux.zip",
```

Comment:
This is bad to maintain as we would be required to constantly update the version strings manually and it can be really easily be forgotten.

Please just link in this case to https://github.com/inexorgame/inexor-core/releases/ without further parameters.

The rest looks good to me 👍